### PR TITLE
Document ecovacs mower cut direction number entity

### PR DESCRIPTION
**Pull request by adsmf**
## Proposed change

Allow configuration of Ecovacs mower cutting direction through new number entity.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/122598
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced a new configuration option for the Ecovacs integration, allowing users to set the mower cutting direction for enhanced control over operations.

### CODE REVIEW

File: source/_integrations/ecovacs.markdown [hunk 1]
```diff
@@ -55,6 +55,7 @@ Additionally, **depending on your model**, the integration provides the followin
   - `Map`: The floorplan/map as an image in SVG format.
 - **Number**:
   - `Clean count`: Set the number of times to clean the area.
+  - `Cut direction`: Set the mower cutting direction.
```
Comment (c0ffeeca7) [inline source/_integrations/ecovacs.markdown]: could you give an example? I don't understand how a number can tell me the direction.

**Comment by c0ffeeca7** [comment]
I'm [drafting this PR](https://developers.home-assistant.io/docs/review-process#prs-are-being-drafted-when-changes-are-needed), as changes are needed. Please undraft it, once the changes have been implemented and the PR is ready for review again.

File: source/_integrations/ecovacs.markdown (hunk 1 above)
Comment (adsmf) [reply source/_integrations/ecovacs.markdown]: Sorry, didn't see the notification requesting a change here.  I've added `(from 0 to 180 degrees)` as that hopefully makes it clearer that this is the the range in direction of travel for the mower.

The terminology I think is hopefully apparent for an owner of the mower where the term "direction" is already used in the Ecovacs app and is accompanied by a little drag arrow across the 180 degree arc for orientation.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @adsmf :+1:

**Pull request merged**